src/components/common/ModalDialog/ModalDialog.tsx

```diff
@@ -21,9 +21,34 @@ export const Modal: FunctionComponent<ModalProp> = ({ toggleOpen, children }) =>
         <div
           className={`flex self-center md:inline-block align-bottom rounded-lg text-left overflow-hidden shadow-xl transform sm:my-8 sm:align-middle sm:max-w-lg sm:w-full`}
         >
-          <div className="bg-white px-4 pt-5 pb-4 sm:p-6 sm:pb-4">{children}</div>
+          <div className="bg-white shadow-2xl px-4 pt-5 pb-4 sm:p-6 sm:pb-4">{children}</div>
         </div>
       </div>
     </div>
   );
 };
+
+interface ConfirmationModalDialogProp {
```

Comment:
What's the diff with ModalDialog

Reply:
Not much different, it is built on top of modal dialog but with a fixed design on title and message style.